app/javascript/pages/browse/Filters.vue

```diff
@@ -30,5 +34,25 @@ export default {
     prop: 'currentFilters',
     event: 'change',
   },
+  methods: {
+    initialOpenStatus(index) {
+      return index === 0
```

Comment:
maybe this is why only first is open?

Reply:
Yup. We can easily change the behavior here to be whatever. 

I thought this was a reasonable first go at it. All expanded looked too cluttered to me. None expanded might lead to people glossing over it and not noticing it's expandable. The first item should be most important, so that seemed an easy, reasonable start